lib/Pathfinder.js

```diff
@@ -0,0 +1,20 @@
+var PF = require('pathfinding');
+
+
+function Pathfinder(map) {
```

Comment:
I don't think this needs to be a class anymore, you could change it to a function like https://github.com/bovard/raid/blob/master/lib/Utils/MoveUtils.js